spec/zenSpec.js

```diff
@@ -0,0 +1,46 @@
+function expect(actual) {
+  return {
+
```

Comment:
I like the way the zenSpec is written, if you want to improve on it you can maybe create a function to change the colour of pass/fail as it is repeated multiple times.